user/pages/05.blog/2018-01-09-docker-improvements/item.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Running OpenNMS Horizon in Docker
+date: 21:48 2018-01-09
+author: Ronny Trommer
+body_classes: header-lite fullwidth blogstyling
+taxonomy:
+    category: docker
+    tag: [docker]
+---
+
+Running applications in containers provide a lot benefits and it's not just a hype.
```

Comment:
Running applications in containers provides many benefits, and it's not just hype.